core/src/main/java/org/apache/carbondata/core/constants/CarbonCommonConstants.java

```diff
@@ -2602,6 +2602,11 @@ private CarbonCommonConstants() {
 
   public static final String FILE_HEADER = "fileHeader";
 
+  @CarbonProperty(dynamicConfigurable = true)
+  public static final String CARBON_OPTIMIZE_FILTER = "carbon.optimize.filter";
```

Comment:
"carbon.reorder.filter" and "carbon.optimize.filter" seem to be doing the same thing. please remove one of them

Reply:
reorder is one part of optimaztion